app.rb

```diff
@@ -1,6 +1,21 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
+    erb(:index)
+  end
```

Comment:
this 'homepage' looks good and is simple from a user perspective (meeting the brief in the simplest way possible).